runtime/drive_handler.go

```diff
@@ -125,10 +125,20 @@ func (h *stubDriveHandler) createStubDrive(driveID, path string) error {
 	return nil
 }
 
+func filterOnlyStubDrives(drives []models.Drive) []models.Drive {
```

Comment:
May be a good idea to document why this occurs which is due to the root drive being always being the last element when `Build` is called on the drive builder from the SDK.

Reply:
You meant, this `Build`.

https://github.com/firecracker-microvm/firecracker-go-sdk/blob/29621e288fa1333809e5b43d867dbb73d82b5456/drives.go#L63

BTW, why do we want to place the root drive at the end?

Reply:
The location of root drive is arbitrary. End was chosen since you didn't need to create a `[]models.Drive` to append to.

Reply:
Thanks. The new revision has some comments around drives initialization. Would it be good enough?